MainModule/Server/Commands/Moderators.lua

```diff
@@ -3142,21 +3142,13 @@ return function(Vargs, env)
 			Function = function(plr,args)
 				for i, v in next,service.GetPlayers(plr,args[1]) do
 					if v.Character then
-						for a, obj in next,v.Character:GetChildren() do
+						for a, obj in next,v.Character:GetDescendants() do
```

Comment:
``						for a, obj in next,v.Character:GetDescendants() do``
You should use
``						for a, obj in next,v.Character:GetChildren() do``.
Using Descendants, yes, may fix a lot of issues, but It would be too powerful. Consider just adding more checks, tools, etc. Using Descendants could be something in another more powerful invis command ("vanish"?) that could also hide you from playerlist, etc.?

Reply:
What you did here breaks it.

Reply:
Yes, because :GetDecendants is kil and has been for ages.

Use :GetChildren(true) as it does the same.

Reply:
Well fuck wrong account but helo